crates/uv/tests/snapshots/ecosystem__transformers-lock-file.snap

```diff
@@ -2884,7 +2884,7 @@ name = "pandas"
 version = "2.2.2"
 source = { registry = "https://pypi.org/simple" }
 dependencies = [
-    { name = "numpy", marker = "python_version == '3.11' or python_version >= '3.12'" },
+    { name = "numpy" },
```

Comment:
This change stands out.. I verified that this expression on it's own simplifies to `python_full_version >= '3.11'`. I guess that it's redundant with the narrowed requires-python in the fork?

Reply:
It looks like the declared metadata is:

```
Requires-Dist: numpy>=1.22.4; python_version < "3.11"
Requires-Dist: numpy>=1.23.2; python_version == "3.11"
Requires-Dist: numpy>=1.26.0; python_version >= "3.12"
```

So I think it's _right_ to remove this marker? (Was it a bug that it was omitted before on Python < 3.11?)